gax-java/gax/src/main/java/com/google/api/gax/retrying/RetrySettings.java

```diff
@@ -101,7 +114,9 @@ public abstract class RetrySettings implements Serializable {
    * Duration.ZERO} and LROs have a default initial poll delay value of {@code
    * Duration.ofMillis(5000)} (5 seconds).
    */
-  public abstract Duration getInitialRetryDelay();
+  public final java.time.Duration getInitialRetryDelayDuration() {
```

Comment:
Same with this

Reply:
As per https://github.com/googleapis/sdk-platform-java/pull/1872#discussion_r1619231111, we use the keyword to ensure the behavior mirrors the `threeten` counterpart that has been implemented